src/cljs/athens/electron.cljs

```diff
@@ -465,6 +521,33 @@
 
 
   (reg-fx
-    :fs/write!
-    (fn []
-      (debounce-write-db true))))
+   :fs/write!
+   (fn []
+     (debounce-write-db true)))
+
+  (reg-fx
+   :invoke-win!
+   (fn [arg-map]
+     (let [channel (:channel arg-map)
+           arg (:arg arg-map)]
```

Comment:
Could use `:keys` destructuring in either the `let` binding or at the `arg-map` parameter.

Reply:
just to complete with an example: `(fn [{:keys [channel arg]}] (if arg (...) (...)))`